### PR TITLE
Configuration.md - Add 'openapi' section

### PR DESCRIPTION
**Pull request by WhiteRabbitDE**
https://github.com/api-platform/core/pull/3837

### CODE REVIEW

File: core/openapi.md [hunk 1]
```diff
@@ -460,3 +460,39 @@ api_platform:
 ```
 
 Note that `clientId` and `clientSecret` are being used by the SwaggerUI if enabled.
+
+
```
Comment (alanpoulain) [inline core/openapi.md:464]:
```suggestion

```

File: core/openapi.md [hunk 2]
```diff
@@ -460,3 +460,39 @@ api_platform:
...
+## Info Object
+
+The [info object](https://swagger.io/specification/#info-object) provides metadata about the API like licensing information or a contact. You can specify this information using Api Platform's configuration:
```
Comment (alanpoulain) [inline core/openapi.md:467]:
```suggestion
The [info object](https://swagger.io/specification/#info-object) provides metadata about the API like licensing information or a contact. You can specify this information using API Platform's configuration:
```

File: core/openapi.md [hunk 3]
```diff
@@ -460,3 +460,39 @@ api_platform:
...
+api_platform:
+    
+    # The title of the API.
+    title: 'API title'
+
+    # The description of the API.
+    description: 'API description'
+
+    # The version of the API.
+    version: '0.0.0'
```
Comment (alanpoulain) [inline core/openapi.md:479]: Do we need to repeat this?

Comment (WhiteRabbitDE) [reply core/openapi.md:479]: We dont need to but its not mentioned on this page and belongs to the info model. I thought that would make the configuration of the Model complete.

File: core/openapi.md (hunk 2 above)
Comment (WhiteRabbitDE) [reply core/openapi.md:467]: This was copied from line 432. Shall i adjust that line as well to have it consistent ?

**Comment by soyuka** [comment]
thanks @WhiteRabbitDE

**Pull request merged**